tests/fixtures/simple_app_with_overrides/packwerk.yml

```diff
@@ -0,0 +1,26 @@
+# See: Setting up the configuration file
+# https://github.com/Shopify/packwerk/blob/main/USAGE.md#setting-up-the-configuration-file
+
+# List of patterns for folder paths to include
+# include:
+# - "**/*.{rb,rake,erb}"
+
+# List of patterns for folder paths to exclude
+# exclude:
+# - "{bin,node_modules,script,tmp,vendor}/**/*"
+
+# Patterns to find package configuration files
+# package_paths: "**/"
+
+# List of custom associations, if any
+# custom_associations:
+# - "cache_belongs_to"
+
+# Whether or not you want the cache enabled (disabled by default)
+cache: false
+
+namespace_overrides:
+  app/services: "::CustomNamespace"
```

Comment:
So does this mean that this affects only `/app/services`? Or all of `packs/*/app/services`? Based on the implementation it looks like the former.

I guess we just want to match the zeitwerk implementation here? So I guess that means any relative folder => its custom namespace.

So if you wanted to mark a whole pack as having a custom namespace, would you do:
```
namespace_overrides:
  packs/bar: "::Bar"
```
?